src/devices/machine/smartmed.cpp

```diff
@@ -455,13 +456,13 @@ void nand_device::command_w(uint8_t data)
 			m_mode = SM_M_INIT;
 			if (!m_write_rnb.isnull())
 			{
-				m_write_rnb( 0);
-				m_write_rnb( 1);
+				m_write_rnb(0);
+				m_write_rnb(1);
 			}
 		}
 		break;
-	/*case 0x11:
-	    break;*/
+		/*case 0x11:
+			break;*/
```

Comment:
The indentation on this and the other commented cases has been messed up (whatever you reformatted with doesn’t realise that it's indented like code and hence the `case` label in the comment is outdated).

Reply:
yeah, I noticed this after I could see it in the brower